### ISSUE TITLE
URL with punctuation won't update

### ISSUE THREAD

**Issue by web1100manager** [samiahmedsiddiqui/custom-permalinks#24]
Lately I've noticed if I find a post on our site that had punctuation in it such as "that's" and then try to update the URL to remove the single quote it will not update. I have tried multiple times and am not sure what is causing it to not update. I've had to go into the database and manually update it there to get it to change

I should also add:
WordPress Version: 5.2.1
PHP Version: 7.2.18

**Comment by samiahmedsiddiqui** [comment]
@web1100manager, firstly permalinks do not contain punctuation when generated. In order to re-create the issue which you highlighted, I created a sample URL with a single quote but the generated permalink didn't include any single quote. Here's the sample title and URL which I generated:
Hello world, how's it going
/hello-world-hows-it-going/

Secondly, if a permalink contains a single quote, it can be updated. I was unable to trace your issue, please share the relevant screenshot of your issue.

**Comment by web1100manager** [comment]
Hi @samiahmedsiddiqui - you are correct that the generated permalink does not include punctuation (I just tested myself). That must have been from one of our content editors that added it in after it was generated. However, after a single quote was added, I am still unable to remove it or edit the permalink. 

Screenshot attached
![Capture](https://user-images.githubusercontent.com/51712453/59301293-42158880-8c5f-11e9-9438-38081fc9a632.PNG)
